rpc_configs/geth.yml

```diff
@@ -34,31 +43,33 @@ methods:
       - name: RewardPercentiles
         ty: Bytes
   # Eth namespace (sign)
-  - method: eth_sign
-    params:
-      - name: Address
-        ty: String
-      - name: Message
-        ty: String
-  - method: eth_signTransaction
-    params:
-      - name: Transaction
-        ty: Bytes
-  # Eth namespace (submit)
-  - method: eth_sendTransaction
-    block_cache:
-      size: 0
-    params:
-      - name: Transaction
-        ty: Bytes
+#  - method: eth_sign
+#    params:
+#      - name: Address
+#        ty: String
+#      - name: Message
+#        ty: String
+#  - method: eth_signTransaction
```

Comment:
These are disabled?

Reply:
See https://www.notion.so/alt-research/CU-table-alt-subway-ff8307541b9e410b99c120c0f6005981 for details